packages/nimble-components/src/table/specs/table-column-specs/table-column-duration-text-hld.md

```diff
@@ -0,0 +1,221 @@
+
+### Blazor integration
+
+A Blazor wrapper will be created for the component. Blazor will not provide a means to access the formatting function of the `DurationFormatter` until there is a client requirement for it.
+
+### Visual Appearance
+
+Will use same text styling as the `nimble-table-column-text`.
+
+## Implementation
+
+In order to format a single value into multiple parts with localized text, we can provide a similar implementation to the current [`ElapsedTimePipe` `transform` method](https://dev.azure.com/ni/DevCentral/_git/Skyline?path=/Web/Workspaces/SystemLinkShared/projects/systemlink-lib-angular/src/pipes/elapsed-time.pipe.ts&version=GBmaster&line=18&lineEnd=19&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents) used in SLE, but instead of using the Angular `DecimalPipe` to format the value, we can just use the native `Intl.NumberFormat`, and pass in the locale we get from the `lang` setting on the `nimble-theme-provider`:
+
+```ts
+const result = [];
+const hours = Math.floor(value / 3600);
+if (hours) {
+    const hoursFormatter = new Intl.NumberFormat(this.lang, { style: 'unit', unit: 'hour' });
+    const formattedHours = hoursFormatter.format(hours);
+    result.push(formattedHours);
+}
+... // do similar work for minutes and seconds
+```
+
+Other implementation details will conform to existing patterns all other table columns handle, such as `CellView` and `GroupHeaderView` implementations.
+
+_Note: For number values that can't be translated into a representable time (e.g. `NaN`, `∞`, `-∞`, negative values), we will simply show a blank cell._
+
+### Considered Corner Cases
+
+#### Really large numbers
+
+At some point numbers that can be formatted as a duration become too large to be practically represented in such a way. We will have a cutoff value (in days) where when the value exceeds that limit, we will revert to displaying the duration as a scientific representation of the value in the seconds unit.
+
+#### Really small numbers
+
+Ultimately, we will expect to be able to configure a duration column to show values down to the nanoseconds unit once we can leverage `Intl.DurationFormat`, and we expose the right formatting APIs on the column. Until then, we should behave as we do for really large numbers, and if the value is below a particular threshold we will format the value in scientific in the seconds unit.
+
+#### Large fractional parts
+
+Currently in SLE, there are corner cases that can result in a display of something like "25 min, 60 sec", which is non-ideal. This happens when the value being formatted has a large fractional part close to 1 (e.g. .999999997). The better behavior would be to either show a formatted value of "26 min", or even "25 min, 59.99 sec", though the former would be a more accurate representation if the value was 59.99999997.
+
+#### Negative Zero
+
+`-0` will simply render as "0".
```

Comment:
nit: either "`-0` will render as "0 sec"" or "`-0` will render the same as 0 does". As this reads now it seems like 0 will render as "0 sec" but -0 will render as "0", which would be confusing.